go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/nmstate/kubernetes-nmstate/api v0.0.0-20250114063637-129b149f6ce9
 	github.com/onsi/ginkgo/v2 v2.22.2
 	github.com/onsi/gomega v1.36.2
-	github.com/openshift-kni/eco-goinfra v0.0.0-20250129204500-1fb7f3c38dc7 // latest
+	github.com/openshift-kni/eco-goinfra v0.0.0-20250130220349-f088d0044ca1 // latest
```

Comment:
shouldn't this be done in a dedicated PR?

Reply:
I don't think so, I do it inside same mr all the time. it has different commit message : "bump vendors"
this is how Nikita explained it too me long time ago